src/components/Chart.tsx

```diff
@@ -162,7 +164,7 @@ function Circle({
     fill: metricCaches.get(metricType)?.colormap.get(d.data.path) ?? "grey",
   })
 
-  return <animated.circle {...props} className={d.data.type} />
+  return <animated.circle {...props} className={d.data.type} onClick={() => (isBlob(d.data))? setClickedBlob(d.data as HydratedGitBlobObject) : setClickedBlob(null)} />
```

Comment:
Awesome change! 🎉
Could you maybe merge the onClick handler with the onClick handler already present on line 78?